website/for_teachers.py

```diff
@@ -985,9 +985,11 @@ def view_adventure(self, user, adventure_id):
             current_page="for-teachers",
         )
 
-    @route("/customize-adventure/<adventure_id>", methods=["GET"])
+    @route("/customize-adventure/<adventure_id>/<current_page>", methods=["GET"])
```

Comment:
i don't understand why you need current_page?

Reply:
it doesn't make sense to have a url like the following:
`"/for-teachers/customize-adventure/view/{{ adventure.id }}/for-teachers"` (for-teachers is repeated twice!!)

Reply:
It's for the go back button, wether to go back to the for-teacher page or to the class page

Reply:
If you're in the customize-adventure page, you can simply go back to the for-teafchers page. If it's necessary, we should design this functionality differently; e.g., save it somewhere and go back accrodingly (in session preferably!)

Reply:
How else should I solve it? I don't understand how it would be possible to use session for it?

Reply:
solved!